mautic/mautic
# [UX] Dashboard date range quick filters

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

Dependency:
- #13770 

This PR adds quick filters for date ranges on the dashboard.
It allows you to select ranges with 1 click instead of manually selecting each date on the calendar.

![image](https://github.com/user-attachments/assets/78a8ed72-caec-4112-b414-eb0a148dc4e9)







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the Dashboard, click on Quick filters and select any of them. See if the range date was automatically applied and all widgets updated.
